include/printer.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+
+namespace hershensh {
+void print(const std::string& line) {
+  std::cout << line << std::endl;
+}
```

Comment:
This should be indented no?

Reply:
Debatable. I feel it won't contribute to the readability, as all code will be in the hersh namespace.
But it's not a strong opinion. What do you think?

Reply:
[Google agrees](https://google.github.io/styleguide/cppguide.html#Namespace_Formatting).